test/node/session-resend-event-on-connection-drop.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const test = require('tap');
+
+const cp = require('child_process');
+const path = require('path');
+
+const jstp = require('../..');
+
+let client = getClient();
+
+const iface = 'iface';
+const event = 'testEvent';
+const ifaces = {
+  iface: {
+    method: (connection, callback) => {
+      client.send(['getSession']);
+      callback(null);
+    },
+  },
+};
+
+const application = new jstp.Application('testApp', ifaces);
+const serverConfig = { applications: [application] };
+const server = jstp.net.createServer(serverConfig);
+
+client.on('message', ([message, ...args]) => {
+  switch (message) {
+    case 'error':
+      test.fail('must not encounter an error');
```

Comment:
You probably forgot to also print the error you are passing in the message.

Reply:
Fixed.

Reply:
@nechaido, it hasn't been fixed yet.